### PR TITLE
Add upgrade case

### PR DESCRIPTION
**Pull request by wewang58**

### CODE REVIEW

File: features/upgrade/build/build-upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      | deploymentconfig=ruby-hello-world |
+
+  # @author wewang@redhat.com
+  # @case_id OCP-13025
+  @upgrade-check
+  @users=upuser1,upuser2
+  Scenario: Check docker and sti build works well after upgrade
+    Given I switch to the first user
+    When I use the "devexp-upgrade" project
+    And status becomes :running of 1 pods labeled:
```
Comment (xiuwang) [inline features/upgrade/build/build-upgrade.feature]: It’s better to trigger a new build for two type builds

Comment (wzheng1) [reply features/upgrade/build/build-upgrade.feature]: agree

Comment (wewang58) [reply features/upgrade/build/build-upgrade.feature]: Updated, please check it again, thanks

File: features/upgrade/build/build-upgrade.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      | project_name | build-upgrade |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | app_repo | openshift/ruby~https://github.com/openshift/ruby-ex |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | app_repo | openshift/ruby~https://github.com/openshift/ruby-hello-world |
+      | strategy | docker                                                       |
+    Then the step should succeed
+    Given I use the "build-upgrade" project
```
Comment (liangxia) [inline features/upgrade/build/build-upgrade.feature]: Looks like we do not need this step.

Comment (wewang58) [reply features/upgrade/build/build-upgrade.feature]: It will report "no projects in cache" when there's no the step

Comment (liangxia) [reply features/upgrade/build/build-upgrade.feature]: Ack

**Pull request merged**